index.html

```diff
@@ -15,15 +15,15 @@
     <div class="container">
 
             <header>
+                    <img class="logo" src="img/lambda-black.png" alt="Lambda Logo">
```

Comment:
For this section, you may want to look into flexbox order.

Reply:
Thanks, I will take a look at that!